react-problems/bubbling-problem/README.md

```diff
@@ -0,0 +1,11 @@
+## Question Description
+
+Render the movie data similar to following:
+
+![Table](https://raw.githubusercontent.com/manmodesanket/content-archives/main/screenshot.png)
+
+After clicking on each cell an alert dialog should popup with respective movie data.
+
+If user clicks on "Imdb Ratings" then an alert dialog should popup with movie rating.
+
```

Comment:
Add ##Solution as a heading & then add the link
Change the title from `CodeSandbox Link` to `<question-name> Solution` & add the link under the whole title